### ISSUE TITLE
Select2 dont init twice

### ISSUE THREAD

**Issue by stephanedemotte** [elliotcondon/acf#659]
Step to reproduce

Add Clone field, focus on the select field (items load)

Change the type to anything else.

Change one more to Clone field

Focus on the select -> boom nothing happen

**Comment by elliotcondon** [comment]
Hi @stephanedemotte 

Thanks for the bug report.
I've just replicated the issue and will hope to have this fixed in 5.7 (the next major release).

For now, you will need to update the field group or reload the page to get the Select2 instance to work again.

Thanks
E

**Comment by stephanedemotte** [comment]
Yep, I have the same bug with a custom select field (this is how I have noticed that) I will wait the update. 

Thanks
